pkg/apis/serving/v1/route_lifecycle.go

```diff
@@ -29,6 +29,9 @@ import (
 var routeCondSet = apis.NewLivingConditionSet(
 	RouteConditionAllTrafficAssigned,
 	RouteConditionIngressReady,
+
+// TODO(nak3):  Add RouteConditionCertificateProvisioned in the next release.
```

Comment:
If we don't plan to do this in this release (I assume 0.14), could you remove the "Release Note" from the description of this PR?

Reply:
Sure, done.

Reply:
Why in the next release?

Reply:
The downgrade test `TestServicePostDowngrade` failed if we added the new condition in `routeCondSet`. The Route became `Unknown` status. (I have confirmed that it does not happen if previous version already has the condition.)
Here is the [failure log](https://prow.knative.dev/view/gcs/knative-prow/pr-logs/pull/knative_serving/7163/pull-knative-serving-upgrade-tests/1236214121729363968)